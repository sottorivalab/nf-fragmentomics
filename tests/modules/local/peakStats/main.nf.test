nextflow_process {

    name "Test Process PEAK_STATS"
    script "modules/local/peakStats/main.nf"
    process "PEAK_STATS"

    test("Should run PEAK_STATS with default parameters") {

        when {
            params {
                outdir = "tests/results"
            }
            process {
                """
                input[0] = [
                    [ caseid:'test', sampleid: 'test', timepoint: 'T1' ],
                    'test_source',
                    [file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/deeptools/test.computeMatrix.mat.gz', checkIfExists: true)]
                ]
                """
            }
        }

        then {
            assert process.success
        }

    }
}
