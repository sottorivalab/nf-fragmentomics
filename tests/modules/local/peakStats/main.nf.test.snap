{
    "Should run PEAK_STATS with default parameters": {
        "content": [
            [
                "versions.yml:md5,cdb1192fff270c9d8e93b9aa3f25bc04"
            ]
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.3"
        },
        "timestamp": "2025-01-22T10:50:48.207795"
    }
}